nio/events/account_data.py

```diff
@@ -67,6 +69,30 @@ def from_dict(cls, event_dict):
         )
 
 
+@attr.s
+class TagEvent(AccountDataEvent):
+    """Informs the client tags of a room. Room tags may include
+        - m.favourite for favourite rooms
+        - m.lowpriority for low priority room
+       A tag may have an optinal order float 0 <= order <=1 for position
```

Comment:
"A tag may have an order between 0 and 1, indicating the room's position towards other rooms with the same tag."